src/main/java/org/opentest4j/MultipleFailuresError.java

```diff
@@ -90,11 +92,13 @@ private static String pluralize(int count, String singular, String plural) {
 		return count == 1 ? singular : plural;
 	}
 
-	private static String nullSafeMessage(AssertionError failure) {
+	private static String nullSafeMessage(Throwable failure) {
+		if (failure == null) {
```

Comment:
If it matters, JUnit4 throws an explicit `NullPointerException` in cases like this. JUnit 5 has an explicit exception for this (`PreconditionViolationException`) which doesn't extend either `IllegalArguementException|`or `NullPointerException`.